celery/app/defaults.py

```diff
@@ -78,6 +78,7 @@ def __repr__(self):
         scheduler=Option('celery.beat:PersistentScheduler'),
         schedule_filename=Option('celerybeat-schedule'),
         sync_every=Option(0, type='int'),
+        cron_starting_deadline_seconds=Option(None, type=int)
```

Comment:
do we have any better naming scheme for this?

Reply:
Kubernetes has a similar option which follows this naming scheme, but I am open to other ideas.

https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#cron-job-limitations

Reply:
what about cron_starting_deadline instead of seconds?

Reply:
Sounds good to me. Updated.